pkg/upgrade/plan/plan.go

```diff
@@ -132,6 +131,9 @@ func RegisterHandlers(ctx context.Context, serviceAccountName, controllerNamespa
 								return obj, err
 							}
 							node.Labels[planLabel] = version
+							if node.Spec.Unschedulable {
```

Comment:
Should we mark the node as schedulable no matter what (as we are doing here) or should we only flip it back if cordschedulableon is true or drain is not nil?